events/api/src/main/java/org/keycloak/events/admin/AuthDetails.java

```diff
@@ -5,44 +5,44 @@
  */
 public class AuthDetails {
 
-    private String realmId;
+    private String authRealmId;
```

Comment:
No need to prefix these variables (and get/setters) with auth, it's in the AuthDetail object so that's implied